src/models/BibDetails.ts

```diff
@@ -363,15 +366,34 @@ export default class BibDetails {
       return null
     }
     const label = "Supplementary content"
-    const values = this.bib.supplementaryContent.map((sc) => {
-      return {
+
+    const hasTOCElectronicResource = this.bib.electronicResources.find(
+      (resource) =>
+        resource.prefLabel &&
+        resource.prefLabel.toLowerCase().includes("table of contents")
+    )
+
+    const isFindingAid = (label: string) => label.includes("finding aid")
+
+    const hasTOCERandTOC = (label: string) =>
+      hasTOCElectronicResource && label.includes("table of contents")
+
+    const values = this.bib.supplementaryContent
```

Comment:
Is there a test for this functionality? I'm still finding it hard to wrap my head around what the expected behavior is supposed to be here. Maybe a good case for an in line comment